mix.exs

```diff
@@ -27,12 +27,12 @@ defmodule Geocoder.Mixfile do
 
   defp deps do
     [{:httpoison, "~> 0.8"},
-     {:poison, "~> 3.0"},
+     {:poison, ">= 0.0.0"},
```

Comment:
I assume you want to completely relax the dependency

Reply:
Sorry, @vyorkin, for not getting back to you earlier. Coming back to your question further up, I think I would prefer `< 4.0.0` in case poison ever starts to submit a changelog.

Reply:
got it, will update in a minute

Reply:
done